php-8.1.yaml

```diff
@@ -49,6 +52,8 @@ environment:
       - sqlite-dev
       - unixodbc-dev
       - wget
+  environment:
+    CFLAGS: -Wno-incompatible-pointer-types
```

Comment:
This implies serious bugs though =/ is there no cherry-pick from upstream to resolve this? is it just a few things or many?

Reply:
We should open maybe an upstream issue ticket, I tried to cherry pick some commits, but could not solve it still